app/controllers/projects_controller.rb

```diff
@@ -19,6 +19,9 @@ def show
       format.swagger do
         render json: Swagger::ProjectDecorator.new(project).to_swagger, content_type: 'application/json'
       end
+      format.api_gateway_integration do
```

Comment:
Rather than using a new MIME type I would use a query parameter on the swagger format, like  `projects/42.swagger?with_api_gateway_integration=true`

It would look like 
```
     format.swagger do
        render json: Swagger::ProjectDecorator.new(project).to_swagger(params[:with_api_gateway_integration]), content_type: 'application/json'
      end
```

Reply:
With keyword argument 

```
     format.swagger do
        swagger = Swagger::ProjectDecorator.new(project).to_swagger(with_api_gateway_integration: params[:with_api_gateway_integration])
        render json: swagger, content_type: 'application/json'
      end
```

Reply:
I tested both and ended with the mime type because I though it would be more consistent with the rest. I revert it. 😄